__main__.py

```diff
@@ -4,43 +4,99 @@
+    )
+
+    parser.add_argument(
+        "-f",
+        "--flush-all",
+        action="store_true",
+        dest="flush_all",
+        help="deletes all subdirectories in the working dir. if this option is used with "
+        + "-w, it will flush the contents of that dir rather than the default working dir.",
+    )
+
+    parser.add_argument(
+        "-d",
+        "--flush-by-date",
+        metavar="YYYY-MM-DD_hh-mm",
+        dest="flush_by_date",
+        help=f"deletes all subdirectories in the working dir older than the arg value. "
+        + "if this option is used with "
+        + "-w, it will flush the contents of that dir rather than the default working dir. "
+        + "argument value must include a year, but all other date specifications are optional. "
+        + "if included, they must follow the YYYY-MM-DD_hh-mm format. for example, the following "
+        + "are valid arg values (seperated by commas): 2024, 2024-07, 2024-07-30_02, "
+        + "2024-07-31_14-53. but the following are invalid args: 202, 2024_14, 07-31, 2024-",
+    )
 
-    parser.add_argument("-f",
-                        "--flush-memory",
-                        action="store_true",
-                        help="clears all .sub files for {test_name}")
+    parser.add_argument(
+        "-p",
+        "--print-tests",
+        action="store_true",
+        dest="print_tests",
+        help="print a list of all the available exerciser tests. if this option is used with "
+        + "-t, it will print the contents of that dir rather than the default tests dir. ",
+    )
 
-    parser.add_argument("-q",
-                        "--query-collector",
-                        action="store_true",
-                        help="queries collector for a list of current ResourceName s in the OSPool. " +
-                             "overwrites any existing .sub files which match the current list, and ignores any which do not")
+    parser.add_argument(
+        "-s",
+        "--snapshot",
+        action="store_true",
+        dest="snapshot",
+        help="query the collector and print a list of currently available resources. ",
+    )
 
-    parser.add_argument("-r",
-                        "--run",
-                        action="store_true",
-                        help="runs the exerciser on {test_name} for any existing submit files. if no submit files exist, does nothing")
+    parser.add_argument(
+        "-w",
+        "--change-working-dir",
+        metavar="new_working_dir",
+        dest="change_working_dir",
+        help="changes the location of the working dir to the specified dir for this run. "
+        + "the arg should be given as str representation of an absolute path to a dir, "
+        + "or a relative path to the Pool_Exerciser root dir. "
+        + "ex: -w ../../dir/subdir/new_working_dir",
+    )
+
+    parser.add_argument(
+        "-t",
+        "--change-test-dir",
```

Comment:
```suggestion
        "--test-dir",
```
Again... `--change` is redundant as the change is implied when specifying a test directory to use